README.md

```diff
@@ -380,6 +380,7 @@ The limited option set includes:
 - `prefix`:   `'''`
 - `suffix`:   `''`
 - `interval`:     `1000`
+- `srearchBody`: `null`
```

Comment:
You said the default was `null` when actuality it's an empty objects

Reply:
Actually, the default here is null: https://github.com/taskrabbit/elasticsearch-dump/blob/a880dfec09122f5a18d1d49f55fde7ea98c3a1c0/bin/elasticdump#L25